cloudtrail.tf

```diff
@@ -20,6 +22,48 @@ resource "aws_cloudtrail" "cloudtrail" {
   tags = "${var.tags}"
 }
 
+#
+# CloudTrail Cloudwatch Log Group
+#
+resource "aws_cloudwatch_log_group" "log_group" {
+  count = "${var.enable_cloudwatch_logs}"
+  name = "CloudTrail/DefaultLogGroup"
+}
+
+#
+# CloudTrail Cloudwatch IAM Role
+#
+data "template_file" "cloudwatch_iam_assume_role_policy_document" {
+  template = "${file("${path.module}/policies/cloudwatch_assume_role_policy.tpl")}"
+}
+
+resource "aws_iam_role" "cloudwatch_iam_role" {
+  count              = "${var.enable_cloudwatch_logs}"
+  name               = "terraform-cloudwatch-role"
+  assume_role_policy = "${data.template_file.cloudwatch_iam_assume_role_policy_document.rendered}"
+}
+
+resource "aws_iam_role_policy_attachment" "cloudwatch_iam_policy_attachment" {
+  count      = "${var.enable_cloudwatch_logs}"
+  role       = "${aws_iam_role.cloudwatch_iam_role.name}"
+  policy_arn = "${aws_iam_policy.cloudwatch_iam_policy.arn}"
+}
+
+data "template_file" "cloudwatch_iam_policy_document" {
+  count    = "${var.enable_cloudwatch_logs}"
+  template = "${file("${path.module}/policies/cloudwatch_policy.tpl")}"
+
+  vars {
+    log_group_arn = "${aws_cloudwatch_log_group.log_group.arn}"
+  }
+}
+
+resource "aws_iam_policy" "cloudwatch_iam_policy" {
+  count  = "${var.enable_cloudwatch_logs}"
+  name   = "terraform-cloudwatch-policy"
```

Comment:
Same here, extract to a variable, with default

Reply:
Same for this one: `terraform-awsconfig-policy`.

Anyway, I'll change it for al configurable stuff in the scope of this PR.